### DISCUSSION TITLE
Why is a transparent window obtained?

### DISCUSSION THREAD

**Discussion by senin-va** [AvaloniaUI/Avalonia#10739] [Q&A] [answered]
I'm trying to use Avalonia 11.0.0-preview1 in Visual Studio 2022 (Version 17.0.6). I'm creating a simple project. When I launch the application, I get a transparent window where nothing is visible.
![image](https://user-images.githubusercontent.com/128593442/226881396-0e133e4b-31d1-48cc-9427-6d49ec04688c.png)

What am I doing wrong?
If I use version 0.10.18 then there is no problem.

**Comment by timunie** [comment, marked as answer]
1. please use at least preview6 if not even latest nightly
2. if you upgrade from 0.10.18, make sure to read the breaking changes page carefully https://github.com/AvaloniaUI/Avalonia/wiki/Breaking-Changes#010--110
